src/extension/extension.ts

```diff
@@ -105,6 +105,30 @@ export async function activate(context: vscode.ExtensionContext) {
       "bazel.copyTargetToClipboard",
       bazelCopyTargetToClipboard,
     ),
+    // URI handler
+    vscode.window.registerUriHandler({
+      async handleUri(uri: vscode.Uri) {
+        try {
+          const workspace = vscode.workspace.workspaceFolders[0];
```

Comment:
presume it's in the first `workspaceFolder`. I suppose something more sophisticated could be done, but not sure it's necessary. Happy to accept suggested change here if someone has a specific suggestion.

Reply:
Without any way of identifying the bazel workspace, I don't think we can do any better than assume the first workspace, so I'm happy with this.